packages/eos-components/src/components/BackToTop.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <transition name="fade">
+    <b-link
+      v-if="hasScrolled"
+      class="back-to-top"
+      @click="goToTop"
+    >
+      <ChevronUpCircle :size="50" />
+    </b-link>
+  </transition>
+</template>
+
+<script>
+  import _ from 'underscore';
+  import ChevronUpCircle from 'vue-material-design-icons/ChevronUpCircle.vue';
+
+  export default {
+    name: 'BackToTop',
+    components: { ChevronUpCircle },
+    data() {
+      return {
+        hasScrolled: false,
+      };
+    },
+    created() {
+      this.debouncedScroll = _.debounce(this.onScroll, 100);
+      window.addEventListener('scroll', this.debouncedScroll);
+    },
+    destroyed() {
+      window.removeEventListener('scroll', this.debouncedScroll);
+    },
+    methods: {
+      goToTop() {
+        window.scrollTo(0, 0);
+      },
+      onScroll() {
+        this.hasScrolled = window.scrollY !== 0;
+      },
+    },
+  }
+</script>
+
+<style lang="scss" scoped>
+  @import '../styles.scss';
+
+  .back-to-top {
+    position: fixed;
+    bottom: 10px;
+    right: 10px;
```

Comment:
Let's use the `$spacer` variable here instead of hardcoding. Doing that ensures that all fits in the grid. Also, adding hardcoded values here and there makes the project very messy and hard to maintain. This comes up a lot in reviews, please remind it.

Reply:
Good catch, I've just replaced that with `$spacer`.